`otp_build -t` crashes with `can't find include lib "xmerl/include/xmerl.hrl"`

**Describe the bug**

Building a tiny release is broken on OTP master.

**To Reproduce**

```
~/src/otp[master]$ git clean -dfx && time MAKEFLAGS=-j8 ERL_TOP=`pwd` ./otp_build all -t
(...)
 ERLC	../ebin/erl2html2.beam
ct_netconfc.erl:68:14: can't find include lib "xmerl/include/xmerl.hrl"
%   68| -include_lib("xmerl/include/xmerl.hrl").
%     |              ^

ct_netconfc.erl:1377:34: record xmlAttribute undefined
% 1377|                                 [#xmlAttribute{name = prolog,
%     |                                  ^

ct_netconfc.erl:1374:5: variable 'Prolog' is unused
% 1374|     Prolog = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>",
%     |     ^

make[4]: *** [../ebin/ct_netconfc.beam] Error 1
make[4]: *** Waiting for unfinished jobs....
cth_conn_log.erl:54:14: can't find include lib "common_test/include/ct.hrl"
%   54| -include_lib("common_test/include/ct.hrl").
%     |              ^

cth_conn_log.erl:103:13: undefined macro 'config/2'
%  103| 			  Dir = ?config(priv_dir,Config),
%     | 			         ^

cth_conn_log.erl:56:2: function pre_init_per_testcase/4 undefined
%   56| -export([init/2,
%     |  ^

make[4]: *** [../ebin/cth_conn_log.beam] Error 1
ct_property_test.erl:40:14: can't find include lib "common_test/include/ct.hrl"
%   40| -include_lib("common_test/include/ct.hrl").
%     |              ^

make[4]: *** [../ebin/ct_property_test.beam] Error 1
make[3]: *** [release] Error 2
make[2]: *** [release] Error 2
make[1]: *** [release] Error 2
make: *** [release] Error 2
MAKEFLAGS=-j8 ERL_TOP=`pwd` ./otp_build all -t  442.52s user 81.90s system 356% cpu 2:27.27 total
```

**Expected behavior**
Build the tiny release.

**Affected versions**
master

--- a/Makefile.in
+++ b/Makefile.in
@@ -399,9 +399,15 @@ ifeq ($(OTP_SMALL_BUILD),true)
 	  ERL_TOP=$(ERL_TOP) PATH=$(INST_PATH_PREFIX)"$${PATH}" \
 	    $(MAKE) TESTROOT="$(RELEASE_ROOT)" release
 else
+ifeq ($(OTP_TINY_BUILD),true)
+	cd $(ERL_TOP)/lib && \
+	  ERL_TOP=$(ERL_TOP) PATH=$(INST_PATH_PREFIX)"$${PATH}" \
+	    $(MAKE) TINY_BUILD=true TESTROOT="$(RELEASE_ROOT)" release
+else
 	cd $(ERL_TOP)/lib && \
 	  ERL_TOP=$(ERL_TOP) PATH=$(INST_PATH_PREFIX)"$${PATH}" \
 	    $(MAKE) BUILD_ALL=1 TESTROOT="$(RELEASE_ROOT)" release
+endif
 endif
 	cd $(ERL_TOP)/erts && \
 	  ERL_TOP=$(ERL_TOP) PATH=$(INST_PATH_PREFIX)"$${PATH}" \

